crates/uv/src/cli.rs

```diff
@@ -51,17 +51,34 @@ pub(crate) struct GlobalArgs {
     #[arg(global = true, action = clap::ArgAction::Count, long, short, conflicts_with = "quiet")]
     pub(crate) verbose: u8,
 
-    /// Disable colors; provided for compatibility with `pip`.
-    #[arg(global = true, long, hide = true, conflicts_with = "color")]
+    /// Disable colors; provided for compatibility with [NO_COLOR](https://no-color.org) and `pip`.
+    /// Takes precedence over `force_color` and `color`.
+    #[arg(
+        global = true,
+        long,
+        hide = true,
+        env = "NO_COLOR",
+        value_parser = clap::builder::BoolishValueParser::new()
+    )]
     pub(crate) no_color: bool,
 
+    /// Forces colors; provided for compatibility with [FORCE_COLOR](https://force-color.org).
+    /// Takes precedence over `color`.
+    #[arg(
+        global = true,
+        long,
+        hide = true,
+        env = "FORCE_COLOR",
+        value_parser = clap::builder::BoolishValueParser::new()
+    )]
+    pub(crate) force_color: bool,
```

Comment:
Implementation wise, I find this dubious to have hidden arguments for pulling in environment variables.  Is this a standard practiced in Ruff?

Reply:
No, in Ruff IIRC we use `colored`. It detects some of these for us, and then we check for `FORCE_COLOR` with:

```rust
// support FORCE_COLOR env var
if let Some(force_color) = std::env::var_os("FORCE_COLOR") {
    if force_color.len() > 0 {
        colored::control::set_override(true);
    }
}
```

Reply:
@samypr100 - Let's change to read these env vars directly, I think?

Reply:
I think i saw a reference to uv using anstream. The equivalent of of `colored` for setting the global is https://docs.rs/anstream/latest/anstream/enum.ColorChoice.html#method.write_global

Reply:
> @samypr100 - Let's change to read these env vars directly, I think?

Done in 1c451f6d44c9e99b8af699e736e36dc09c17c27f

Reply:
Thanks Ed!